applications/FluidDynamicsApplication/symbolic_generation/navier_stokes/symbolic_stokes_cpp_template.cpp

```diff
@@ -0,0 +1,369 @@
+}
+
+template <class TElementData>
+void SymbolicStokes<TElementData>::AddBoundaryTraction(
+    TElementData& rData,
+    const Vector& rUnitNormal,
+    MatrixType& rLHS,
+    VectorType& rRHS)
+{
+    // Set the current Gauss pt. Voigt notation normal projection matrix
+    BoundedMatrix<double, Dim, StrainSize> voigt_normal_projection_matrix = ZeroMatrix(Dim, StrainSize);
+    FluidElementUtilities<NumNodes>::VoigtTransformForProduct(rUnitNormal, voigt_normal_projection_matrix);
+
+    // Set the current Gauss pt. strain matrix
+    BoundedMatrix<double, StrainSize, LocalSize> B_matrix = ZeroMatrix(StrainSize, LocalSize);
+    FluidElementUtilities<NumNodes>::GetStrainMatrix(rData.DN_DX, B_matrix);
+
+    // Compute some Gauss pt. auxiliar matrices
+    const BoundedMatrix<double, Dim, StrainSize> aux_matrix_AC = prod(voigt_normal_projection_matrix, rData.C);
+    const BoundedMatrix<double, StrainSize, LocalSize> aux_matrix_ACB = prod(aux_matrix_AC, B_matrix);
+
+    // Fill the pressure to Voigt notation operator matrix
+    BoundedMatrix<double, StrainSize, LocalSize> pres_to_voigt_matrix_op = ZeroMatrix(StrainSize, LocalSize);
+    for (unsigned int i=0; i<NumNodes; ++i) {
+        for (unsigned int comp=0; comp<Dim; ++comp) {
+            pres_to_voigt_matrix_op(comp, i*BlockSize+Dim) = rData.N[i];
+        }
+    }
+
+    // Set the shape functions auxiliar transpose matrix
+    BoundedMatrix<double, LocalSize, Dim> N_aux_trans = ZeroMatrix(LocalSize, Dim);
+    for (unsigned int i=0; i<NumNodes; ++i) {
+        for (unsigned int comp=0; comp<Dim; ++comp) {
+            N_aux_trans(i*BlockSize+comp, comp) = rData.N[i];
+        }
+    }
+
+    // Contribution coming fron the shear stress operator
+    noalias(rData.lhs) = prod(N_aux_trans, aux_matrix_ACB);
+
+    // Contribution coming from the pressure terms
+    const BoundedMatrix<double, LocalSize, StrainSize> N_voigt_proj_matrix = prod(N_aux_trans, voigt_normal_projection_matrix);
+    noalias(rData.lhs) -= prod(N_voigt_proj_matrix, pres_to_voigt_matrix_op);
+
+    array_1d<double,LocalSize> values;
+    this->GetCurrentValuesVector(rData,values);
+
+    rData.lhs *= rData.Weight;
+    noalias(rLHS) -= rData.lhs;
+    noalias(rRHS) += prod(rData.lhs,values);
+}
+
+template <>
+void SymbolicStokes< SymbolicStokesData<2,3> >::ComputeGaussPointLHSContribution(
+    SymbolicStokesData<2,3>& rData,
+    MatrixType& rLHS)
+{
+    const double rho = rData.Density;
+    const double mu = rData.EffectiveViscosity;
+
+    const double h = rData.ElementSize;
+    const double dt = rData.DeltaTime;
+    const double bdf0 = rData.bdf0;
+    const double dyn_tau = rData.DynamicTau;
+
+    // Get constitutive matrix
+    const Matrix& C = rData.C;
+
+    // Get shape function values
+    const auto& N = rData.N;
+    const auto& DN = rData.DN_DX;
+
+    // Stabilization parameters
+    constexpr double stab_c1 = 4.0;
+    constexpr double stab_c2 = 2.0;
```

Comment:
`stab_c2` will be unused, so probably can be removed.

Reply:
I don't think so... The `stab_c2` constant multiplies the pressure subscale.

Reply:
c2 is always multiplied by the norm of the convective velocity so it should be not used in stokes element I think.

Reply:
@rubenzorrilla can you please merge master into this branch before merging this PR, just to be safe => #7239 
thx

Reply:
> c2 is always multiplied by the norm of the convective velocity so it should be not used in stokes element I think.

Sorry. You are both right, my last comment was referring to tau_2. If we remove the second term in the stabilization constant as @mrhashemi points it wouldn't be required any more. This gives control on the convective term so makes no sense to keep it in the Stokes element.  I'll remove it.

Reply:
> @rubenzorrilla can you please merge master into this branch before merging this PR, just to be safe => #7239
> thx

Sure. I'll do it before adding all the code review suggestions.

Reply:
`stab_c2` removed and `master` merged. Thanks.